core/src/main/java/org/jboss/pnc/build/finder/core/BuildFinder.java

```diff
@@ -595,11 +596,15 @@ private KojiBuild findBestBuildFromCandidates(List<KojiBuild> candidates, List<K
                     buildCache.put(cacheRpmBuildInfo.getBuildInfo().getId(), cacheRpmBuildInfo);
                 }
             } else {
-                if (cacheManager == null || (cacheArchiveInfos = checksumCaches.get(ChecksumType.md5)
-                        .get(checksum.getValue())) == null) {
+                ListKojiArchiveInfoProtobufWrapper wrapper = null;
```

Comment:
Could you add s pace before an after the `if` block? I always like to have blank lines around all blocks. I wish the formatter could to things liek this automatically but it's not specific enough in terms of whitepace.

Reply:
done